File: python_programming/Practices/fizzbuzz.py
#CC 7th fizzbuzz

n = 1 

while n <51:
    if  n % 15 ==0:
        print("fizzbuzz")
    elif n % 5 == 0:
        print("buzz")
    elif n % 3 == 0:
        print("fizz")
    else:
        print(f"{n}")
    n += 1

File: c_programming/practices/fizzbuzz.c
//CC 7th fizzbuzz
#include <stdio.h>

int main(void){

    int num= 1;
    while(num<51){
        num++;
        if(num%15){
            printf("Fizzbuzz\n");
        }else if(num%5){
            printf("buzz\n");
        }else if(num%3){
            printf("fizz\n");
        }else{
            printf("%d\n", num);
        } 
    }


    return 0;
}